Add Mapper's Choice Awards 2024 news post (#13006)
ppy/osu-wiki: 2025-03-26-mappers-choice-awards-2024.md

---
layout: post
title: Mapper's Choice Awards 2024
date: 2025-03-26 01:30:00 +0000
---

### Get Ready to Cast Your Vote in the Mapper’s Choice Awards 2024!  

> It’s time to step up, vote, and crown last year’s best Ranked maps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*  

Reviewer: ```suggestion
It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*  
```

![](link for banner)  

For those unfamiliar with the event, the **Mapper’s Choice Awards** (*MCA*) is a **community-hosted competition** where mappers nominate and vote for the best maps and mappers across various categories, celebrating mapping excellence in all its forms.

[Just like last year](https://osu.ppy.sh/home/news/2024-02-01-mappers-choice-awards-2023), **winners in each category will receive special awards!** To learn more, keep reading below and check out the [Corsace website](https://corsace.io/), where you can find past MCA results and more details about the event.  

---

## Mapper’s Choice Awards 2024  

> If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognize the most outstanding maps of the year!  

Reviewer: ```suggestion
If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps of the year!  
```

### **Key Changes for 2024**  
- The number of voting categories has been **reduced**.  
- osu!supporter rewards will now be granted **only** for the **Grand Award** categories.  

As always, MCA consists of three stages: **nomination, voting, and results.** Check the schedule below for key dates and deadlines!  

- **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition.*  
- **Voting:** *Eligible users vote on the nominated submissions.*  
- **Results:** *Winners are determined by voting, with a livestream revealing the podium finishers in each category.*  

If you’re eligible, head over to the [**Mapper’s Choice Awards 2024**](https://mca.corsace.io/2024/) website to get started!

---  

### **Schedule**  

> | Event | Date (UTC) |  

Reviewer: ```suggestion
| Event | Date (UTC) |
```

| --: | :-- |  

> | **Nomination phase** | 2025-03-26 – 2025-03-31 00:00 |  

Reviewer: ```suggestion
| **Nomination phase** | 2025-03-26 – 2025-03-31 00:00 |
```

Reviewer: i would recommend putting UTC format time there

> | **Voting phase** | 2025-04-03 – 2025-04-08 00:00 |  

Reviewer: ```suggestion
| **Voting phase** | 2025-04-03 – 2025-04-08 00:00 |
```

Reviewer: same here

> | **Results stream** (on the [Corsace Twitch channel](https://www.twitch.tv/corsace)) | 2025-04-14 |  

Reviewer: ```suggestion
| **Results stream** [Corsace Twitch channel](https://www.twitch.tv/corsace) | 2025-04-14 |
```

Reviewer: and here

> | **Final results** (on the [Corsace website](https://mca.corsace.io/2024/)) | 2025-04-15 |  

Reviewer: ```suggestion
| **Final results** [Corsace website](https://mca.corsace.io/2024/) | 2025-04-15 |
```

Reviewer: and of course here aswell

---

### **Rewards**  

MCA 2024 features exciting rewards for the winners of each category:  

- **osu!supporter**  
  - Grand Award winners: **4 months**  
- **Badges**  
  - Special badges for **Grand Award** category winners.  
  - Generic badges for **regular category** winners. 

---

### **A Year in Mapping (AYIM) Update**  

> Unfortunately, **A Year in Mapping (AYIM)** will not be hosted this year. *As the main organizers have stepped away from the project, it will no longer be supported.* We want to extend our **heartfelt gratitude** to those who contributed over the years, making AYIM a cherished part of the community.  

Reviewer: ```suggestion
Unfortunately, **A Year in Mapping (AYIM)** will not be hosted this year. *As the main organisers have stepped away from the project, it will no longer be supported.* We want to extend our **heartfelt gratitude** to those who contributed over the years, making AYIM a cherished part of the community.  
```

Now that you’re caught up on all the details, it’s time for the most important part—**nominating and voting!** [**Visit the website to get started!**](https://mca.corsace.io/2024/)  

> —Maxus, momoyo  

Reviewer: ```suggestion
—Maxus & momoyo  
```
